Tests/Unit/Document/DocumentTests.swift

```diff
@@ -733,6 +733,14 @@ class DocumentTests: XCTestCase {
             XCTAssertEqual((event as? LocalChangeEvent)?.value.operations.compactMap { $0.path }, ["$", "$.", "$..obj", "$..obj", "$..obj", "$..obj", "$."])
         }
 
+        await target.subscribe(targetPath: "$.") { _ in
+            Task {
+                let array = try? await target.getValueByPath(path: "$.") as? JSONObject
+
+                XCTAssertTrue(array != nil)
```

Comment:
Can we verify if the value is properly returned instead of checking if the array is not nil?
The `getValueByPath()` method is used to retrieve values, such as `doc.getValueByPath('$.obj.num')`, instead of using `doc.getRoot().obj.num`. Therefore, it would be better to verify if the value is properly returned.

(Refer to the example using `getValueByPath` in the js-sdk:
https://github.com/yorkie-team/yorkie-js-sdk/blob/395d052f2a6a3c1a5cef794a2e45b0312a63e945/public/multi.html#L169 )

Reply:
I added some Test code that verifies the value of getValueByPath() results.

Reply:
Thank you for the modification! 
I have a question: Is it better to silently ignore the usage of a key containing `.` instead of explicitly throwing an error?

```swift
root["$$...hello"] = [] as [Any]

// The key name can't contain "."
XCTAssertTrue(root["$$...hello"] == nil)
```

Reply:
I will add an assert and modify it so that you can know explicitly whether the wrong value was used in the development.